src/main/java/org/gridsuite/securityanalysis/server/util/LimitReductionConfig.java

```diff
@@ -0,0 +1,36 @@
+package org.gridsuite.securityanalysis.server.util;
```

Comment:
Move in the package `services` ?
Configuration is a spring bean !

Reply:
done in https://github.com/gridsuite/security-analysis-server/pull/134/commits/058536f7c4805eb40ee4f7933ca2789a3871ef5f